reps/2023-07-20-torch-trainer-apis.md

```diff
@@ -102,46 +143,33 @@ trainer = LightningTrainer(
 trainer.fit()
 ```
 
-**`TorchTrainer` (Proposed):**
+#### `TorchTrainer` (Proposed):
 
 In this proposal, we provide a few common utilties that the user can use directly in their training code to configure the `Trainer` object.
 
 - `get_devices` - Returns a list of devices to use for training.
-- `prepare_trainer` - Validates that the `Trainer` object is configured correctly to be compatible with Ray Train.
+- `prepare_trainer` - Validates and/or makes modifications so that the `Trainer` object is configured correctly to be compatible with Ray Train.
 - `RayDDPStrategy`, `RayFSDPStrategy`, `RayDeepSpeedStrategy` - `LightningStrategy`s for different training strategies that are compatible with Ray Train.
 - `RayEnvironment` - A `LightningEnvironment` that is compatible with Ray Train.
-- `RayModelCheckpoint` - A `LightningCallback` that configures the `Trainer` to report metrics and checkpoints to Ray Train.
 
 With this, the user can directly interact with the Lightning interface, and is able define their training logic to use additional functionality such as `lightning.Trainer.test()`.
 
 ```python
 import lightning
+import ray.train.lightning
 from ray.train.torch import TorchTrainer
-from ray.train.lightning import (
-    get_devices, 
-    prepare_trainer,
-    RayDDPStrategy, 
-    RayEnvironment, 
-    RayModelCheckpoint
-)
 
 def train_func(config):
 
     ...
 
-    devices = get_devices()
-    strategy = RayDDPStrategy()
-    environment = RayEnvironment()
-    checkpoint_callback = RayModelCheckpoint()
-
     lightning_trainer = lightning.Trainer(
-        devices=devices,
-        strategy=strategy,
-        plugins=[environment],
-        callbacks=[checkpoint_callback],
+        devices=ray.train.lightning.get_devices(),
+        strategy=ray.train.lightning.RayDDPStrategy(),
+        plugins=[ray.train.lightning.RayEnvironment()],
```

Comment:
```suggestion
        plugins=[ray.train.lightning.RayEnvPlugin()],
```

Reply:
Hmm this is a subclass of the `LightningEnvironment` class which is a [Plugin](https://pytorch-lightning.readthedocs.io/en/2.0.6/extensions/plugins.html#cluster-environments). 

What about `RayLightningEnvironment`?